apps/server/src/shared/domain/types/learnroom.types.ts

```diff
@@ -1,11 +1,15 @@
 export enum LearnroomTypes {
 	'Course' = 'course',
+	'Lesson' = 'lesson',
 }
 
-export type LearnroomMetadata = {
+export type BaseMetadata = {
```

Comment:
The idea here is good. unfortunately, LearnroomTypes is a trap in this case.
(Uwe already fell into it and I failed to notice in the first version of the share service).

LearnroomTypes is used in the Rooms overview (= the course overview, like https://niedersachsen.cloud/rooms-overview) and denotes all entities that can be shown on that page. Lesson is not such an entity (but back then we expected teams to become one). So though these types appear to be the right ones to build upon, we have a false duplicate here.

Reply:
BaseMetadata removed